pkgs/by-name/st/styluslabs-write3/package.nix

```diff
@@ -0,0 +1,112 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  gitMinimal,
+  pkg-config,
+  SDL2,
+  xorg,
+  dbus,
+  libcap,
+  libGL,
+  roboto,
+  imagemagick,
+  makeDesktopItem,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "Write3";
```

Comment:
Where does this name come from? We should just use `styluslabs-write`.

The old drv should be renamed to stylus-labs-bin.

The app should just be called Write like upstream intends.